src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -1388,43 +1351,63 @@ ExecuteModifyTasksWithoutResults(List *taskList)
  * returns 0.
  */
 int64
-ExecuteModifyTasksSequentiallyWithoutResults(List *taskList, CmdType operation)
+ExecuteModifyTasksSequentially(CitusScanState *scanState, List *taskList,
+							   CmdType operation, bool hasReturning)
 {
 	ListCell *taskCell = NULL;
 	bool multipleTasks = list_length(taskList) > 1;
-	bool expectResults = false;
 	int64 affectedTupleCount = 0;
-	bool alwaysThrowErrorOnFailure = true;
+	bool alwaysThrowErrorOnFailure = false;
 	bool taskListRequires2PC = TaskListRequires2PC(taskList);
 
-	/* decide on whether to use coordinated transaction and 2PC */
+	/*
```

Comment:
Just changed `alwaysThrowErrorOnFailure` to match `RouterSequentialModifyExecScan()`.